index.js

```diff
@@ -158,7 +158,7 @@
 
   //  trimTrailingSpaces :: String -> String
   var trimTrailingSpaces = function(s) {
-    return s.replace(/[ ]+$/, '');
+    return s.replace(/[ ]+$/gm, '');
   };
```

Comment:
Trimming all trailing spaces *after* formatting a message simplifies matters as we no longer need to clean up as we go.